### DISCUSSION TITLE
Exit AR event

### DISCUSSION THREAD

**Discussion by danilokrulj** [google/model-viewer#3457] [Q&A] [answered]
Hi everyone, does anyone know if there is an exit-ar event i can listen to? I have a model in AR and i can change its height (position.y) but upon exiting the AR session the viewer remembers the y position the model was in before exiting the session. I would like to reset the y position upon exiting the session.

I know i can make a custom exit ar button and than listen for events on that, but i would like to go with the default one :)

**Comment by danilokrulj** [comment, marked as answer]
`modelViewer.shadowRoot.querySelector('#default-exit-webxr-ar-button').addEventListener("click", function());`

this is the way to go, in case anyone else gets stuck here :)

**Reply by elalish** [reply]
I think you'd be better off listening to [`ar-status`](https://modelviewer.dev/docs/index.html#entrydocs-augmentedreality-events-arStatus): exit should fire "not-presenting". Anything in our Shadow root is not in our public API, so you might get hit with breaking changes.

**Reply by PaoloDivision5** [reply]
What do you suggest doing when using QuickLook for ios, and the event of exiting is needed?

**Reply by elalish** [reply]
Unfortunately, I don't think QuickLook notifies us of this at all.
